uber/site_sections/registration.py

```diff
@@ -502,6 +502,19 @@ def lost_badge(self, session, id):
         raise HTTPRedirect('index?message={}', 'Badge has been recorded as lost.')
 
     @ajax
+    def check_out_attendee_in_error(self, session, **params):
```

Comment:
I'm not sure why you're using ``**params`` here instead of just having the arguments be ``(self, session, id)``, since that's all that would ever get passed here and it makes the code below somewhat more awkward.